src/components/publiq-ui/Label.js

```diff
@@ -6,11 +6,14 @@ const StyledLabel = styled(Form.Label)`
   font-weight: 700;
 `;
 
-const Label = ({ children, className }) => (
-  <StyledLabel className={className}>{children}</StyledLabel>
+const Label = ({ id, children, className }) => (
+  <StyledLabel htmlFor={id} className={className}>
```

Comment:
Shouldn't we just keep the htmlFor prop for this? `id` looks as if you're providing the label an id, which is not the case